modules/ROOT/pages/tools/neo4j-admin/unbind.adoc

```diff
@@ -5,6 +5,16 @@
 
 You can use the `neo4j-admin server unbind` command to remove and archive the cluster state of a cluster server so that it can rebind to a cluster.
 
+[WARNING]
+====
+Avoid running the `unbind` command on all servers, as this results in the loss of cluster quorum.
+
+In Neo4j 5.x, use the `unbind` command only when troubleshooting a specific server and you need to convert it into a standalone instance.
```

Comment:
You don't have to want to convert it to a standalone instance (I know this was said by field in the chat). I would just merge this sentence with the next. Like

In Neo4j 5.x, use the `unbind` command only when troubleshooting a specific server and keep in mind there is no guarantee that the allocator will reassign the same databases to this server, potentially resulting in orphaned database stores.
The `unbind` command preserves all database stores on the server; and when the unbound server is restarted and enabled, it is seen as an entirely new server.